packages/app/src/cli/models/extensions/specifications/editor_extension_collection.ts

```diff
@@ -17,17 +18,33 @@ const EditorExtensionCollectionSchema = BaseSchema.extend({
   type: zod.literal('editor_extension_collection'),
 })
 
+export type EditorExtensionCollectionType = zod.infer<typeof EditorExtensionCollectionSchema>
+
 const editorExtensionCollectionSpecification = createExtensionSpecification({
   identifier: 'editor_extension_collection',
   schema: EditorExtensionCollectionSchema,
   appModuleFeatures: (_) => [],
+  validate: async (config, path) => {
+    const errors: string[] = []
+    const inCollection = makeExtensionsInCollection(config)
+
+    if (!config.handle) {
+      return err(`Editor extension collection with name ${config.name} must have a handle`)
```

Comment:
We'll need the length validator on core, but it would be nice to have it here too since these validations fire before it tries to deploy, and it's such an easy validation.

Reply:
I believe handle already has a length validator by default

Reply:
Oh nice, and only 30 characters!

Reply:
You don't need this validation. instead of using `BaseSchema` use `BaseSchemaWithHandle`, then `zod` will take care of making sure there is a handle always.